app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -1098,6 +1108,14 @@ struct DeviceDelegateOculusVR::State {
       controller->SetButtonState(controllerState.index, ControllerDelegate::BUTTON_TOUCHPAD, 0, trackpadPressed, trackpadTouched);
 
       controller->SetAxes(controllerState.index, axes, kNumAxes);
+
+      if (renderMode == device::RenderMode::Immersive) {
+        if (triggerPressed) {
+          controller->SetSelectActionStart(controllerState.index);
+        } else {
+          controller->SetSelectActionStop(controllerState.index);
+        }
+      }
```

Comment:
Do we need to stop the action if we exit immersive mode and the user is still pressing one of these action buttons?

Reply:
Yes! we should. I have given an update for this. Please let me know if I miss something.